DNN Platform/Library/Common/Utilities/Config.cs

```diff
@@ -318,8 +318,7 @@ public static long GetRequestFilterSize(IApplicationStatusInfo appStatus)
 
             if (httpNode != null)
             {
-                var maxAllowedContentLength = XmlUtils.GetAttributeValueAsLong(httpNode.CreateNavigator(), "maxAllowedContentLength", 30000000);
-                return maxAllowedContentLength / 1024 / 1024;
+                return 4294967295 / 1024 / 1024; // 4GB (max allowedContentLength supported in IIS7)

```

Comment:
This value used to be dynamic, and is now hardcoded. Should this be adjusting the default on line 311, instead?

Reply:
So before this change, it would send the currently configured maximum, preventing users from changing the value (in the UI) to anything larger (if on IIS7). With this change it allows using the maximum permitted in IIS7

Reply:
Yeah, what he said ☝🏼

Reply:
In other words if you had manually configured for let's say 100MB, then in the UI you could only change the value to one between 28MB and 100MB

Reply:
It looks to me like this method was returning the correct value, but that [the usage by `SecurityController`](https://github.com/dnnsoftware/Dnn.Platform/blob/40b2bf5b63ec8c15e8f241cd385b469fec11ea11/Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Services/SecurityController.cs#L713) was incorrectly using is as the maximum supported size, which is not true. It seems to me like we should use this value in `SecurityController` and leave `Config.GetRequestFilterSize` as-is. Am I missing what we're trying to do here?

Reply:
Oh, I understand, makes a lot of sense...

Reply:
@bdukes agreed - a commit was just pushed.